tests/testcases.py

```diff
@@ -33,7 +33,7 @@ def setUp(self):
         constants.CONFIG_PATH = self.temp_config_path
         write_default_config()
         save_config_value(constants.CONFIG_SPECS_REPO_KEY, 'github.com/org/dusty-specs')
-        override_repo(get_specs_repo(), self.temp_specs_path)
+        override_repo(get_specs_repo().remote_path, self.temp_specs_path)
```

Comment:
Oh `override_repo` you mean? It has a lot of app logic in it (config mutation, log to client, etc.) that would pollute the Repo class.